vars/gradleWrapper.groovy

```diff
@@ -9,10 +9,13 @@ def cleanupCommand(String command) {
  * execute gradlew or gradlew.bat based on current os
  */
 def call(String command, Boolean returnStatus = false, Boolean returnStdout = false) {
+
+    def logLevel = params.LOG_LEVEL ?: (env.LOG_LEVEL ?: "quiet")
+
     if(isUnix()) {
-        return sh(script: "./gradlew ${cleanupCommand(command)}", returnStdout:returnStdout, returnStatus:returnStatus)
+        return sh(script: "./gradlew --${logLevel.toLowerCase()} ${cleanupCommand(command)}", returnStdout:returnStdout, returnStatus:returnStatus)
     }
     else {
-        return bat(script: "gradlew.bat ${cleanupCommand(command)}", returnStdout:returnStdout, returnStatus:returnStatus)
+        return bat(script: "gradlew.bat --${logLevel.toLowerCase()} ${cleanupCommand(command)}", returnStdout:returnStdout, returnStatus:returnStatus)
```

Comment:
I would move the loglevel logic into a method like `cleanupCommand` and would return the formated `--${logleve}` string or an empty string when nothing is provided in case gradle is changeing the default log level.

Reply:
Very well.